app/helpers/profile_helper.rb

```diff
@@ -2,4 +2,12 @@ module ProfileHelper
   def edit_profile_button
     link_to t(:edit_profile), edit_user_path, class: 'btn btn-success'
   end
+
+  def cancel_edit_profile_button
+    link_to t(:cancel), :back, class: 'btn btn-default' if current_user.profile_completed?
+  end
```

Comment:
Would changing this to

```suggestion
  def cancel_edit_profile_button
    link_to t(:cancel), :user_path, class: 'btn btn-default' if current_user.profile_completed?
  end
```
fix this very small https://github.com/mumuki/mumuki-laboratory/issues/1483 issue?

Or would it break some redirection stuff or use case I'm unaware of?